### PR TITLE
feat(eekb_agent): agents of finding drafts by given author and by author of question

### PR DESCRIPTION
**Pull request by MikhailSadovsky**

### CODE REVIEW

File: ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_author/agent_of_finding_all_drafts_of_author.scsi [hunk 1]
```diff
@@ -0,0 +1,317 @@
+agent_of_finding_all_drafts_of_author
+=> nrel_main_idtf:
+	[агентная scp-программа поиска всех черновиков разработчикa] (* <- lang_ru;; *);
+	[agent scp-program of finding all drafts of author] (* <- lang_en;; *);
+	<- agent_scp_program;;
+
+scp_program -> agent_of_finding_all_drafts_of_author
+	(*
+	-> rrel_params: .agent_of_finding_all_drafts_of_author_params
```
Comment (ShunkevichDV) [inline ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_author/agent_of_finding_all_drafts_of_author.scsi]: No need to name  .agent_of_finding_all_drafts_of_author_params, use ...

File: ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_author/agent_of_finding_all_drafts_of_author.scsi [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+
+scp_program -> agent_of_finding_all_drafts_of_author
+	(*
+	-> rrel_params: .agent_of_finding_all_drafts_of_author_params 
+		(*
+		-> rrel_1: rrel_in: _event;;
+		-> rrel_2: rrel_in: _input_arc;;
+		*);;
+
+	-> rrel_operators: .agent_of_finding_all_drafts_of_author_operator_set
```
Comment (ShunkevichDV) [inline ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_author/agent_of_finding_all_drafts_of_author.scsi]: No need to name .agent_of_finding_all_drafts_of_author_operator_se, use ...

File: ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_author/agent_of_finding_all_drafts_of_author.scsi [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+			<- searchElStr3;;
+
+			-> rrel_1: rrel_assign: rrel_scp_var: _temp;;
+			-> rrel_2: rrel_fixed: rrel_scp_var: _input_arc;;
+			-> rrel_3: rrel_assign: rrel_scp_var: _quest;;
+
+			=> nrel_goto: .agent_of_finding_all_drafts_of_author_operator1B;;
+			*);;
+
+		-> .agent_of_finding_all_drafts_of_author_operator1B
```
Comment (ShunkevichDV) [inline ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_author/agent_of_finding_all_drafts_of_author.scsi]: Use .. in operator names like ..agent_of_finding_all_drafts_of_author_operator1B

File: ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_author/agent_of_finding_all_drafts_of_author.scsi (hunk 1 above)
Comment (MikhailSadovsky) [reply ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_author/agent_of_finding_all_drafts_of_author.scsi]: Updates were done

File: ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_author/agent_of_finding_all_drafts_of_author.scsi (hunk 2 above)
Comment (MikhailSadovsky) [reply ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_author/agent_of_finding_all_drafts_of_author.scsi]: Updates were done

File: ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_author/agent_of_finding_all_drafts_of_author.scsi (hunk 3 above)
Comment (MikhailSadovsky) [reply ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_author/agent_of_finding_all_drafts_of_author.scsi]: Updates were done

**Pull request merged**